Mobile Buy SDK/Mobile Buy SDK Tests/BUYClientTest.m

```diff
@@ -74,7 +75,7 @@ - (NSData *)dataForCartFromClient:(BUYClient *)client
 {
 	BUYCart *cart = [self cart];
 	BUYCheckout *checkout = [[BUYCheckout alloc] initWithCart:cart];
-	NSURLSessionDataTask *task = [self.client createCheckout:checkout completion:nil];
+	NSURLSessionDataTask *task = [self.client createCheckout:checkout completion:^(BUYCheckout *checkout, NSError *error) {}];
```

Comment:
Precisely, since the callback is the only way for the functions to return data, they make no sense if a callback isn't provided.